yaks/lib/yaks/behaviour/optional_includes.rb

```diff
@@ -0,0 +1,29 @@
+require "rack/utils"
+
+module Yaks
+  module Behaviour
+    module OptionalIncludes
+      RACK_KEY = "yaks.optional_includes".freeze
+
+      def associations
+        super.select do |association|
+          association.if != Undefined || include_association?(association)
```

Comment:
@groyoh Yes, but this is exactly what I want. I want that if someone specifies `:if`, that `include_association?` checking is ignored. Because I want it to mean that their overriding this behaviour for that association. If someone specifies `:if -> { false }`, that should mean that they never want this association included (in which case of course they wouldn't even put it there).